SWLOR.Game.Server/CustomEffect/DivineCellEffect.cs

```diff
@@ -21,7 +21,7 @@ public string Apply(NWCreature oCaster, NWObject oTarget, int effectiveLevel)
         public void Tick(NWCreature oCaster, NWObject oTarget, int currentTick, int effectiveLevel, string data)
         {
             if (currentTick % 2 != 0) return;
-            int damage = RandomService.D4(1) + (oCaster.RightHand.DamageBonus / 5);
+            int damage = RandomService.D4(1) + (oCaster.RightHand.DamageBonus / 10);
```

Comment:
This one is 10 and the others are 8. Intentional?

Reply:
Yes, Divine bypasses our DI% with stuff. I